cfme/tests/cloud/test_providers.py

```diff
@@ -1215,52 +1215,51 @@ def test_refresh_with_stack_without_parameters(provider, request, stack_without_
     provider.validate_stats(ui=True)
 
 
-@test_requirements.ec2
-@pytest.mark.manual
-def test_ec2_public_images():
+@test_requirements.cloud
+@pytest.mark.long_running
+@pytest.mark.provider([AzureProvider, EC2Provider], scope="function")
+def test_public_images_enable_disable(setup_provider, request, appliance, provider):
     """
+    Bugzilla:
+        1612086
+        1491330
```

Comment:
Is this test providing coverage for these BZs? If so can you add them as `automates`, i.e. `@pytest.mark.meta(automates=[1612086, 1491330])`?

Reply:
Added